docs/src/pages/index.tsx

```diff
@@ -53,6 +53,7 @@ export default function Home() {
           <div
             css={css`
               margin-top: 3rem;
+              font-size: 1rem;
```

Comment:
I assume this can be removed once you implement `MdxLayout.tsx`/`DocLayout.tsx`.

Reply:
Also the margin on h1s.